src/prefect/tasks/dbt/dbt.py

```diff
@@ -407,12 +413,6 @@ def run(
                 for link, name in artifact_links:
                     markdown += f"- [{name}]({link})\n"
 
```

Comment:
@desertaxle you still need to actually call the create_markdown_artifact method:
```python
                markdown = f"Artifacts for dbt Cloud run {run['id']} of job {job_id}\n"
                for link, name in artifact_links:
                    markdown += f"- [{name}]({link})\n"
                from prefect.backend.artifacts import create_markdown_artifact
                create_markdown_artifact(markdown)
```